From 5387e06ef87bb5895e9b3dbade615538c7b4547d Mon Sep 17 00:00:00 2001
From: maurerr <mariusd84@gmail.com>
Date: Tue, 12 Feb 2019 14:33:35 +0100
Subject: [PATCH] not std for round

---
 src/cryfs-cli/Cli.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/cryfs-cli/Cli.cpp b/src/cryfs-cli/Cli.cpp
index 8a80a4d..835d9fc 100644
--- a/src/cryfs-cli/Cli.cpp
+++ b/src/cryfs-cli/Cli.cpp
@@ -273,7 +273,7 @@ namespace cryfs {
         if (minutes == none) {
             return none;
         }
-        uint64_t millis = std::round(60000 * (*minutes));
+        uint64_t millis = round(60000 * (*minutes));
         return make_unique_ref<CallAfterTimeout>(milliseconds(millis), callback);
     }
 
-- 
2.11.0

