From 7c1d36b7dc9ef03d324c9e8279daa77dac2b2798 Mon Sep 17 00:00:00 2001
From: maurerr <mariusd84@gmail.com>
Date: Mon, 11 Feb 2019 09:46:54 +0100
Subject: [PATCH] defined llroundl function

---
 src/hyperloglog.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/hyperloglog.c b/src/hyperloglog.c
index ba3a3ab..2a03858 100644
--- a/src/hyperloglog.c
+++ b/src/hyperloglog.c
@@ -33,6 +33,10 @@
 
 #include <stdint.h>
 #include <math.h>
+long long int llroundl (double x)
+{
+	  return llround (x);
+}
 
 /* The Redis HyperLogLog implementation is based on the following ideas:
  *
-- 
2.11.0

