From 5401820da205748acd1a03e8aea2129bbce50b85 Mon Sep 17 00:00:00 2001
From: maurerr <mariusd84@gmail.com>
Date: Mon, 11 Feb 2019 15:01:25 +0100
Subject: [PATCH] define MAP_HUGETLB in core_helpers.c

---
 src/src/core_helpers.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/src/core_helpers.c b/src/src/core_helpers.c
index e9b7780..cb962b6 100644
--- a/src/src/core_helpers.c
+++ b/src/src/core_helpers.c
@@ -64,7 +64,7 @@
 #include "mem_find.h"
 #include "core_helpers.h"
 
-
+#define MAP_HUGETLB 0x0040000
 
 void *
 mmalloc_fd(uintptr_t fd, size_t size) {
-- 
2.11.0

