From 32f18b78b30a83303dee702a18b7bc10653b7414 Mon Sep 17 00:00:00 2001
From: maurerr <mariusd84@gmail.com>
Date: Sun, 10 Feb 2019 15:16:46 +0100
Subject: [PATCH] fix missing backtrace

---
 src/error.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/error.c b/src/error.c
index 7738eed..4dd4f77 100644
--- a/src/error.c
+++ b/src/error.c
@@ -35,7 +35,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <signal.h>
-#if defined(__GLIBC__)
+#if !defined(__UCLIBC__)
 #include <execinfo.h>
 #endif
 #include <sys/types.h>
@@ -148,7 +148,7 @@ sigsegv_handler (int sig)
   FILE *fp = stderr;
   int pid = getpid ();
 
-#if defined(__GLIBC__)
+#if !defined(__UCLIBC__)
   char **messages;
   size_t size, i;
   void *trace_stack[TRACE_SIZE];
@@ -161,7 +161,7 @@ sigsegv_handler (int sig)
 
   dump_struct (fp);
 
-#if defined(__GLIBC__)
+#if !defined(__UCLIBC__)
   size = backtrace (trace_stack, TRACE_SIZE);
   messages = backtrace_symbols (trace_stack, size);
 
-- 
2.11.0

